.eslintrc.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable import/unambiguous */
```

Comment:
disabled a rule to make our eslintrc file more readable

### Before

<img width="651" alt="Screenshot 2024-12-03 at 2 20 32 PM" src="https://github.com/user-attachments/assets/a381c681-ab26-4fc5-a30c-954b1c15b3d9">

### After

<img width="633" alt="Screenshot 2024-12-03 at 2 20 51 PM" src="https://github.com/user-attachments/assets/5682a1c7-d62d-441b-a979-6e22dba62e67">